packages/catalog-readmodel-writer/src/consumerServiceV2.ts

```diff
@@ -10,7 +10,11 @@ export async function handleMessageV2(
   message: EServiceEventEnvelopeV2,
   eservices: EServiceCollection
 ): Promise<void> {
-  logger.info(message);
+  logger.info(
+    JSON.stringify(message, (_, v) =>
+      typeof v === "bigint" ? v.toString() : v
+    )
+  );
```

Comment:
FYI: The bigint problems with the logger has been solved in [this PR](https://github.com/pagopa/interop-be-monorepo/pull/273)